lib/autoupdate/agent/updater.go

```diff
@@ -575,6 +597,9 @@ func (u *Updater) Status(ctx context.Context) (Status, error) {
 	if err := validateConfigSpec(&cfg.Spec, OverrideConfig{}); err != nil {
 		return out, trace.Wrap(err)
 	}
+	if cfg.Spec.Proxy == "" {
+		return out, ErrNotInstalled
+	}
```

Comment:
This might be confusing, according to the updater's update.yaml, teleport is not installed. However, teleport might be:
- installed via the package manager
- installed by another teleport-update in a different namespace

I don't think we have to detect all those use-cases (although it would provider very actionable output for the user), but we might want to change our wording and say that there are no teleport-update-managed teleport installation.

Reply:
Adjusted